### ISSUE TITLE
Template loop bug for table

### ISSUE THREAD

**Issue by WesWrench** [clokep/mwcomposerfromhell#5]
The following code:
```
== Firewall appliances ==
{| class="wikitable sortable" style="font-size: 85%; text-align: center; width: auto;"
|-
! Firewall
! [[Software license|License]]
! Cost
! [[Operating system|OS]]
|-
! [[Clavister]]
| {{Proprietary}}
| Included on all Clavister<br>NGFWs
| Proprietary operating system cOS Core
|-
! [[Check Point]]
| {{Proprietary}}
| Included on Check Point<br>security gateways
| Proprietary operating system [[Check Point IPSO]]<br>and Gaia ([[Linux]]-based)
|-
! [[Fortinet#FortiGate Firewall Platform|FortiGate]]
| {{Proprietary}}
| Included on all Fortigate<br>devices
| Proprietary, FortiOS,
Based on the [[Linux]] kernel
|-
! [[Palo Alto Networks]]
| {{Proprietary}}
| Included on Palo Alto<br>Networks firewalls
| Proprietary, PAN-OS,
Based on the [[Linux]] kernel
|-
! [[Sophos]]
| {{Proprietary}}
| Included on Sophos UTM
| [[Linux]]-based appliance
|-
! [[Sourcefire|Cisco ASA Firepower]]
| {{Proprietary}}
| Included on all CISCO<br>ASA devices
| Proprietary operating system.
Based on the [[Linux]] kernel.
|-
! [[Cisco PIX]]
| {{Proprietary}}
| Included on all CISCO<br>PIX devices
| Proprietary operating system
|-
! [[Juniper Networks|Juniper]] SSG
| {{Proprietary}}
| Included on Netscreen<br>security gateways
| Proprietary operating system [[ScreenOS]]
|-
! [[Juniper Networks|Juniper]] SRX
| {{Proprietary}}
| Included on SRX<br>security gateways
| Proprietary operating system [[Junos]]
|-
! [[SonicWall]]
| {{Proprietary}}
| Included on Dell appliance
| Proprietary operating system [[SonicOs|SonicOS]]
Based on the [[Linux]] kernel
|-
! [[Barracuda Networks|Barracuda]] Firewall
| {{Proprietary}}
| Included Firewall Next Generation appliance
| [[Windows]]-based appliance<br>embedded firewall distribution
|-
! [[Cyberoam]]
| {{Proprietary}}
| Included Firewall Sophos appliance
| [[Windows]]-based appliance<br>embedded firewall distribution
|-
! [[D-Link]]
| {{Proprietary}}
| Included Firewall DFL
| [[Windows]]-based appliance<br>embedded firewall distribution
|-
! [[Endian Firewall]]
| {{Proprietary}}
| Free / Paid
| [[Linux]]-based appliance
|-
! [[Forcepoint]] NGFW
| {{Proprietary}}
| Included on all Forcepoint NGFW devices
| Proprietary operating system
|-
! [[OPNsense]]
| {{open source|[[Simplified BSD license|Simplified BSD / FreeBSD License]]}}
| Free / Paid
| [[FreeBSD]]-based appliance<br>firewall distribution
|-
! [[pfSense]]
| {{open source|[[Apache 2.0]] / [[Proprietary]] (Plus)}}
| Free / Paid
| [[FreeBSD]]-based appliance<br>firewall distribution
|-
! [[Zeroshell]]
| {{open source|[[GPL]]}}
| Free / Paid
| [[Linux]]/[[NanoBSD]]-based appliance<br>firewall distribution
|-
! [[SmoothWall]]
| {{open source|[[GPL]]}}
| Free / Paid
| [[Linux]]-based appliance<br>embedded firewall distribution
|-
! [[IPFire]]
| {{open source|[[GPL]]}}
| Free (Donations welcomed)
| [[Linux]]-based appliance<br>embedded firewall distribution
|-
! [[WatchGuard]]
| {{Proprietary}}
|Included on all Firebox devices
|Proprietary, Fireware OS,
Based on the [[Linux]] kernel
|-
! [[WinGate]]
| {{Proprietary}}
| Free / Paid
| [[Windows]]-based appliance<br>embedded firewall distribution 
|}
```

returns

```
<p><span class="error">Template loop detected: <a href="/wiki/Template:Proprietary" title="Template:Proprietary">Template:Proprietary</a></span>
</p>
```

**Comment by clokep** [comment]
This seems to be working fine on current `main`, but no in `0.4`. I'm assuming you're using the 0.4 release?

**Comment by clokep** [comment]
Probably 4cc37cbb894ee66b676104eebb7e94f9970c6a64?

**Comment by clokep** [comment]
This should be fixed in v0.5.
